insurance/data_generation.ipynb

```diff
@@ -64,6 +64,20 @@
     "    os.environ[\"AWS_BUCKET\"] = \"mm-workflow\""
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!sudo curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -\n",
```

Comment:
Can this installation  be done based on DB_PROVIDER?

Reply:
Yes it can be done but we are importing packages unconditionally, earlier i did that but import was failing, so for that i need to import conditionally also

Reply:
Addressed this